graph/graph.go

```diff
@@ -341,7 +332,7 @@ func (graph *Graph) ByParent() (map[string][]*image.Image, error) {
 			byParent[parent.ID] = []*image.Image{img}
 		}
 	})
-	return byParent, err
+	return byParent, nil
```

Comment:
Make sense. The error can be only in case of problems with reading graph directory.